src/main/java/com/pretz/dyntreeserver/controller/AuthController.java

```diff
@@ -32,4 +33,9 @@ public AuthController(UserService userService) {
         userService.createUser(user);
         return new ResponseEntity<>("User " + user.getName() + " successfully created!", new HttpHeaders(), HttpStatus.CREATED);
     }
+
+    @RequestMapping(method = GET, path = "/get_all_users")
+    public ResponseEntity<String> getAllUsers() {
+        return new ResponseEntity<>(userService.getAllUsers().toString(), new HttpHeaders(), HttpStatus.OK);
```

Comment:
Why `toString`? Shouldn't Jackson be able to map `List<User>` to a JSON Array?

Reply:
It is a placeholder. In [this task](https://trello.com/c/O1vi0s4I/19-discuss-api-to-be-compliant-with-json-api-standard) we will create unified API compliant with JSON API guidelines. So right now it will stay as it is. :)

Reply:
placeholder or not, it'll be almost impossible to parse it by the client service

Reply:
[Fixed here](https://github.com/Elm0xz/dyn-tree-server/commit/de8464b2f49c4b09dfa9289d0c0747c1078fa528)